src/keyutil/keyutil.c

```diff
@@ -314,6 +315,38 @@ static int verb_public(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
+static int verb_extract_certificate(int argc, char *argv[], void *userdata) {
+        _cleanup_(X509_freep) X509 *certificate = NULL;
+        int r;
+
+        if (!arg_certificate)
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "--certificate= must be specified");
```

Comment:
I'd like to add full stop at the end.
(Though, the same error message without period is used at many places. Feel free to ignore.)